.github/workflows/rerun-workflows.yml

```diff
@@ -19,17 +19,78 @@ jobs:
           echo "View the failed run attempt (#${{ github.event.workflow_run.run_attempt }}) using the following link:" >> $GITHUB_STEP_SUMMARY
           echo "${{ github.event.workflow_run.html_url }}" >> $GITHUB_STEP_SUMMARY
 
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - run: npm install @slack/web-api
       - name: Trigger a re-run
         uses: actions/github-script@v6
         with:
           script: |
             const MAX_ATTEMPTS = 2;
             const ATTEMPT = ${{ github.event.workflow_run.run_attempt }};
 
+            const breaker = `@${context.actor}`;
+            const breaking_commit = context.sha;
+            const branch = context.ref;
+
             if (ATTEMPT <= MAX_ATTEMPTS) {
               github.rest.actions.reRunWorkflowFailedJobs({
                 owner: context.repo.owner,
                 repo: context.repo.repo,
                 run_id: ${{ github.event.workflow_run.id }},
               });
+            } else {
+              // notify slack of repeated failure
+              const WebClient = require('@slack/web-api');
+              const slack = new WebClient(${{ secrets.SLACK_BOT_TOKEN }});
+
+              await slack.chat.postMessage({
+                channel: 'engineering',
```

Comment:
maybe this should be `engineering-ci` ? I just don't want people to be able to leave that channel and ignore 😂